satellite/metabase/rangedloop/service_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2022 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package rangedloop_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+
+	"storj.io/storj/satellite/metabase/rangedloop"
+	"storj.io/storj/satellite/metabase/rangedloop/rangedlooptest"
+	"storj.io/storj/satellite/metabase/segmentloop"
+)
+
+func TestLoop(t *testing.T) {
+	for _, parallelism := range []int{1, 2, 3} {
+		for _, nSegments := range []int{0, 1, 2, 11} {
+			for _, nObservers := range []int{0, 1, 2} {
+				t.Run(
+					fmt.Sprintf("TestLoop_par%d_seg%d_obs%d", parallelism, nSegments, nObservers),
```

Comment:
I think we don't need `TestLoop_` prefix as this name will part of parent test. Now it will be `TestLoop/TestLoop_par1_seg0_obs0`.

Reply:
Done